### ISSUE TITLE
Custom Fields Not Attached to User Entity

### ISSUE THREAD

**Issue by skp3zl** [drupal-graphql/graphql#560]
In beta6, fields added to the User entity (via /admin/config/people/accounts/fields) do not appear in the generated schema. Default User fields, such as uid and mail, show up fine.

```
{
  currentUserContext {
    ... on User {
      uid
      # Doesn't exist in schema
      fieldTeam{
        targetId
      }
    }
  }
}
```

**Comment by achton** [comment]
@sebkamil Is this working for you with latest dev version? I cannot get access to the custom user fields in GraphiQL whatever I do...

**Comment by skp3zl** [comment]
@achton this was fixed in 1b1eb9d which hasn't been pushed to drupal.org yet. You can apply the patch if you want to test it out.

It's also possible to add this repo to composer if you want the latest and greatest going forward.
